ansible/roles/ml-analytics-service/templates/config.j2

```diff
@@ -0,0 +1,157 @@
+[ML_APP_NAME]
+
+survey_app = {{ ml_analytics_survey_app_name }}
+
+integrated_app = {{ ml_analytics_integrated_app_name }}
+
+[API_HEADERS]
+
+content_type = application/json
+
+authorization = {{ ml_analytics_api_authorization_key }}
```

Comment:
why are we having this file in the DevOps repo? we should have this file in the code repo, else when the dev updates the code, or variable, we don't have to do it multiple places...

Reply:
yes rajesh you right right, i will remove from here, let it be in code repo

Reply:
done